services/cron-service/env-defaults

```diff
@@ -12,3 +12,9 @@ WORK_REAPER_BATCH_SIZE=2000
 
 # The cron schedule for restarting Prometheus if it is down
 RESTART_PROMETHEUS_CRON="*/10 * * * *" # every 10 minutes
+
+# The cron schedule for the user-work updater
+USER_WORK_UPDATER_CRON="*/30 * * * *" # every thirty minutes
+
+# Rows in user-work older than this will cause the user-work for a job to be recalculated
+USER_WORK_UPDATE_AGE="1 HOUR"
```

Comment:
I am lazy, so I don't quite see the point of making this configuration parameter value that we control so complicated. Why not just call it something like `USER_WORK_EXPIRATION_MINUTES` with a value type of number?

Reply:
Then we can delete most of the code in this PR.

Reply:
I would prefer a numeric value in minutes over a string as well.

Reply:
done